### ISSUE TITLE
Pipeline raw value check fails with negative integers

### ISSUE THREAD

**Issue by CaptChrisD** [rrrene/credo#322]
### Environment

* Credo version (`mix credo -v`):  "0.5.3", "0c405b36e7651245a8ed63c09e2d52c2e2b89b6d02b1570c4d611e0fcbecf4a2"
* Erlang/Elixir version (`elixir -v`): 1.3.4
* Operating system: OSX

### What were you trying to do?

```elixir
start =
  -1
  |> now_plus(:day)
  |> Ecto.DateTime.to_date
  |> Ecto.Date.to_string
```

### Expected outcome

No warning

### Actual outcome

Warning for pipeline starting without raw value

But

```elixir
start =
  1
  |> now_plus(:day)
  |> Ecto.DateTime.to_date
  |> Ecto.Date.to_string
```

Does not produce a warning

**Comment by rrrene** [comment]
Ouch, thanks for reporting! :+1:
